README.md

```diff
@@ -1,7 +1,242 @@
+```rust
+use transistor::docker::{Action};
+use transistor::client::Crux;
+use transistor::types::{CruxId};
+
+let person = Person {
+    crux__db___id: CruxId::new("hello-entity"), 
+    first_name: "Hello".to_string(), 
+    last_name: "World".to_string()
+};
+
+let client = Crux::new("localhost", "3000").docker_client();
+
+let document = client.document_by_id(tx_body.db___content_hash).unwrap();
+// Person {
+//     crux__db___id: CruxId(
+//         ":hello-entity",
+//     ),
+//     first_name: "Hello",
+//     last_name: "World",
+// }
+```
+
+* `documents` resquests endpoint `/documents` via `POST`. The argument of this reuqest is a vector of `content-hashes` that converts to an edn set as a body. Returns a map of document ids and respective documents for a given set of content hashes submitted in the request body.
+```rust
+use transistor::docker::{Action};
+use transistor::client::Crux;
+use transistor::types::{CruxId};
+
+let person1 = Person {
+    crux__db___id: CruxId::new("hello-entity"), 
+    ...
+};
+
+let person2 = Person {
+    crux__db___id: CruxId::new("hello-documents"), 
+    ...
+};
+
+let client = Crux::new("localhost", "3000").docker_client();
+
+let contesnt_hashes = vec![tx_body1.db___content_hash, tx_body2.db___content_hash];
+
+let documents = client.documents(contesnt_hashes).unwrap();
+// {
+//     "1828ebf4466f98ea3f5252a58734208cd0414376": Map(
+//         Map(
+//             {
+//                 ":crux.db/id": Key(
+//                     ":hello-entity",
+//                 ),
+//                 ":first-name": Str(
+//                     "Hello",
+//                 ),
+//                 ":last-name": Str(
+//                     "World",
+//                 ),
+//             },
+//         ),
+//     ),
+//     "1aeb98a4e11f30827e0304a9c289aad673b6cf57": Map(
+//         Map(
+//             {
+//                 ":crux.db/id": Key(
+//                     ":hello-documents",
+//                 ),
+//                 ":first-name": Str(
+//                     "Hello",
+//                 ),
+//                 ":last-name": Str(
+//                     "Documents",
+//                 ),
+//             },
+//         ),
+//     ),
+// }
+```
+
+`Action` is an enum with a set of options to use in association with the function `tx_log`:
+* [`PUT`](https://opencrux.com/docs#transactions-put) - Write a version of a document
+* [`Delete`](https://opencrux.com/docs#transactions-delete) - Deletes the specific document at a given valid time
+* [`Evict`](https://opencrux.com/docs#transactions-evict) - Evicts a document entirely, including all historical versions (receives only the ID to evict)
+
+## Dependencies
+A strong dependency of this crate is the [edn-rs](https://crates.io/crates/edn-rs) crate, as many of the return types are in the [Edn format](https://github.com/edn-format/edn). The sync http client is `reqwest` with the feature `blocking`.
```

Comment:
What do you think of changing the order of `feature` and `blocking` on the end of the line? Idk if this is nitty picking, if you think it is good as it is, just dismiss the comment 🙂